nav2_dwb_controller/dwb_core/include/dwb_core/publisher.hpp

```diff
@@ -103,7 +110,7 @@ class DWBPublisher
   bool publish_cost_grid_pc_;
 
   // Previously published marker count for removing markers as needed
-  int prev_marker_count_;
+  unsigned prev_marker_count_;
```

Comment:
unsigned int?

Reply:
Our coding standard is silent on this. In the ros 2 source tree, I find it used with int 810 times and without 748 times. 

Seems there is no precedent. 

Do you want to try to standardize this for our project or leave it up to the individual?

Reply:
I'm OK with leaving it as is.